server/db/migrations/20211230165827_harden-letter_versions-table.js

```diff
@@ -0,0 +1,44 @@
+const tableName = 'letter_versions'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.alterTable(tableName, function (table) {
+      // Rename columns
+      table.renameColumn('campaignid', 'campaign_id')
+
+      // Add NOT NULL constraints to simple fields
+      table.integer('campaign_id').unsigned().notNullable().alter()
+      table.string('template_id').notNullable().alter()
+
+      // Add columns
+      table.string('municipality')
+
+      // Add indexes
+      table.index(['campaign_id'])
+
+      // Add unique indexes
+      table.unique(['template_id'])
```

Comment:
Is unique required here? I think we can add it later if needed.

Reply:
My understanding is that they should always be unique, so this gives us uniqueness enforcement + indexing. 🤷🏻‍♂️